src/main/java/com/faforever/client/preferences/DataPrefs.java

```diff
@@ -37,6 +37,10 @@ public Path getBinDirectory() {
     return getBaseDataDirectory().resolve(BIN_SUB_FOLDER);
   }
 
+  public Path getReplayBinDirectory() {
+    return getBaseDataDirectory().resolve(CACHE_SUB_FOLDER).resolve(BIN_SUB_FOLDER);
```

Comment:
We should not be using the cache folder for in use files

Reply:
So use the replay folder again?

Reply:
more likely just a new replayBin folder